cmd/eskip/args_test.go

```diff
@@ -84,7 +84,7 @@ func TestProcessArgs(t *testing.T) {
 		false,
 		nil,
 		[]*medium{{
-			typ: etcd,
+			typ:        etcd,
```

Comment:
can you drop that please or run gofmt on it?

Reply:
This was actually gofmt doing it.